python/src/pipeline/pipeline.py

```diff
@@ -39,10 +40,12 @@
 import urllib
 import uuid
 
+import cloudstorage
```

Comment:
Can we make this a PyPi dependency and import it as such?

Example from the MR code...
PyPi dependency: https://github.com/GoogleCloudPlatform/appengine-mapreduce/blob/master/python/src/setup.py
Import: https://github.com/GoogleCloudPlatform/appengine-mapreduce/blob/master/python/src/mapreduce/tools/gcs_file_seg_reader.py#L11

I am not thrilled about how it is imported right now, but I'd like to be consistent with the MR idiom since pipeline and MR usually run side-by-side.